config/config.exs

```diff
@@ -27,7 +27,7 @@ config :phoenix, :json_library, Jason
 
 config :digital_collex, DigitalCollex.ElasticsearchCluster,
   # The URL where Elasticsearch is hosted on your system
-  url: "http://localhost:9201",
+  url: System.get_env("ELASTIC_SEARCH_URL", "http://localhost:9201"),
```

Comment:
Changing the default port value here to 9200 did not get the build to pass: https://circleci.com/gh/samvera-labs/digital_collections_elixir_example/43 (on a different branch)